geopandas/io/tests/test_file.py

```diff
@@ -766,6 +767,24 @@ def test_read_file__ignore_all_fields(engine):
     assert gdf.columns.tolist() == ["geometry"]
 
 
+def test_read_file__where_filter(engine):
+    skip_pyogrio_not_supported(engine)  # pyogrio has "columns" keyword instead
```

Comment:
`pyogrio` has a [`where` filter](https://pyogrio.readthedocs.io/en/latest/api.html#pyogrio.read_dataframe) that should match what is being added for Fiona here; only need to skip if pyogrio is not available.

Reply:
Thanks 👍